api/src/main/java/org/killbill/billing/subscription/api/SubscriptionBaseInternalApi.java

```diff
@@ -93,9 +93,7 @@ public SubscriptionBaseBundle createBundleForAccount(UUID accountId, String bund
 
     public void updateExternalKey(UUID bundleId, String newExternalKey, InternalCallContext context);
 
-    public Iterable<DateTime> getFutureNotificationsForAccount(InternalCallContext context);
-
-    public Map<UUID, DateTime> getNextFutureEventForSubscriptions(final SubscriptionBaseTransitionType eventType, final InternalCallContext internalCallContext);
+    public Map<UUID, DateTime> getNextFutureEventForSubscriptions(final List<SubscriptionBaseTransitionType> eventTypes, final InternalCallContext internalCallContext);
```

Comment:
Is this still in use?

Reply:
It's not, i thought of removing it, and somehow proceeded to not do it. The `SubscriptionDao#getFutureEventsForAccount` methods is also unused. Will remove both.